browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -161,6 +176,26 @@ void BraveTabContextMenuContents::ExecuteBraveCommand(int command_id) {
       brave::ToggleVerticalTabStripFloatingMode(browser_);
       return;
     }
+    case BraveTabMenuModel::CommandToggleTabMuted: {
+      auto* model = static_cast<BraveTabStripModel*>(controller_->model());
+      auto indices = model->GetTabIndicesForCommandAt(tab_index_);
+      std::vector<content::WebContents*> contentses;
+      std::transform(
+          indices.begin(), indices.end(), std::back_inserter(contentses),
+          [&model](int index) { return model->GetWebContentsAt(index); });
+
+      auto all_muted = std::all_of(contentses.begin(), contentses.end(),
+                                   [](content::WebContents* contents) {
+                                     return contents->IsAudioMuted();
+                                   });
+
+      for (auto* contents : contentses) {
+        chrome::SetTabAudioMuted(contents, !all_muted,
+                                 TabMutedReason::AUDIO_INDICATOR,
+                                 std::string());
```

Comment:
very optional: It'd be helpful if we have param name here, to be sure about empty arugment.

```suggestion
                                 /* extension_id = */ std::string());
```